kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/manager/ResourceManager.java

```diff
@@ -78,6 +76,10 @@ public static ExtensionContext getTestContext() {
         return testContext;
     }
 
+    private static String getContextUniqueName() {
+        return getTestContext().getRequiredTestClass().getName() + "." + getTestContext().getDisplayName();
```

Comment:
As I said in my previous comment, the display name is not necessarily unique.

Reply:
Using the test method name won't guarantee uniqueness.  Think about parameterised tests.  (@ParameterizedTest).. same test method, n test cases.

Reply:
Having the combination of `testClass + test method` seems unique to me. The name of a class shall be unique and the name of a test method into the class shall be unique as well

Reply:
I'll change it, but it won't be a problem as we need to remove all the resources after each test execution and they won't clash.

Reply:
Anyway `uniqueId` seems to be pretty similar to the solution I gave, but maybe it will change when using @TestTemplate or @ParameterizedTest: `[engine:junit-jupiter]/[class:io.kroxylicious.systemtests.OperatorST]/[method:operatorInstallation()]`